### ISSUE TITLE
Subunit display

### ISSUE THREAD

**Issue by laurahaley** [City-of-Bloomington/master_address#13]
When listing addresses results that also list the number of units (like the search by street screen). Can we make sure that only the number of current subunits are listed.

**Comment by inghamn** [comment]
We might want to be more nuanced.  Retired addresses in the search results show zero subunits, even though they have subunits.  The subunits are retired when an address is retired.  That seems incongruous as well.

**Comment by inghamn** [comment]
Would it be correct to display the count of subunits that have the same status as the address?

**Comment by laurahaley** [comment]
Yes, that sounds good. If the address was current, only the count of
current subunits would appear. If the address was retired, the count of
retired subunits would appear.

Laura Haley
GIS Manager
City of Bloomington, IN
Information & Technology Services
812.349-3599
haleyl@bloomington.in.gov
http://bloomington.in.gov/gis
